### ISSUE TITLE
Redirects for outdated page

### ISSUE THREAD

**Issue by rebloor** [mozilla/extension-workshop#1368]
[PR #1364](https://github.com/mozilla/extension-workshop/pull/1364) is removing various pieces of outdated content. 
To avoid landing any external links to these pages onto a 404 error could the following redirects be added:

- https://extensionworkshop.com/documentation/develop/developing-extensions-for-firefox-for-android-fennec/ to https://extensionworkshop.com/documentation/develop/developing-extensions-for-firefox-for-android/
- https://extensionworkshop.com/documentation/develop/porting-a-legacy-firefox-extension/ to https://extensionworkshop.com/documentation/develop/
- https://extensionworkshop.com/documentation/develop/comparison-with-the-add-on-sdk/ to https://extensionworkshop.com/documentation/develop/
- https://extensionworkshop.com/documentation/develop/comparison-with-xul-xpcom-extensions/ to https://extensionworkshop.com/documentation/develop/

**Comment by eviljeff** [comment]
@rebloor is this an up-to-date list of URLs that need redirects? @bqbn will look into adding them to the CDN

**Comment by rebloor** [comment]
@eviljeff it is

**Comment by rebloor** [comment]
@eviljeff any progress on this one?

**Comment by eviljeff** [comment]
> @eviljeff any progress on this one?

@bqbn was adding them to the CDN but I don't know if he's been able to do it yet

**Comment by bqbn** [comment]
No, I have not had a chance to look into this issue. As Andrew and I discussed last time, we may be able to do the redirection in Cloudfront but the idea needs to be tested.

**Comment by rebloor** [comment]
Thanks @bqbn I will look forward to an update in due course.

**Comment by rebloor** [comment]
@bqbn any update on this?

**Comment by bqbn** [comment]
I just filed https://github.com/mozilla/extension-workshop/pull/1563 for this.

However, have we explore the option of letting the old pages return 404? If not too many fennec browsers are out there for example, why do we want to maintain a redirection for it? Even though redirecting a few pages is OK, if we are adding more pages to the list in the future, it will become clumsy to manage and maintain. Before we decide to redirect a page, I hope it has been discussed and those pages are really necessary to stay for a long period of time.

**Comment by rebloor** [comment]
@bqbn at the start of this process I asked whether there were any stats we could review to see what, if any, traffic these pages got. Unfortunately, there doesn't appear to be anything we can use. Therefore, the safest course seemed the redirection route. (It's also the practice we follow on MDN.)

**Comment by willdurand** [comment]
fixed by https://github.com/mozilla/extension-workshop/pull/1563
